lib/ddtrace/contrib/sinatra/tracer_middleware.rb

```diff
@@ -55,6 +55,16 @@ def call(env)
               # resource; if no resource was set, let's use a fallback
               span.resource = env['REQUEST_METHOD'] if span.resource.nil?
 
+              rack_request_span = env[Datadog::Contrib::Rack::Ext::RACK_ENV_REQUEST_SPAN]
+              if rack_request_span
```

Comment:
Minor, but is there a more concise way of expressing this logic? `rack_request_span.resource ||= span.resource if rack_request_span`?

Reply:
Indeed. This is a carryover from a conflict, there used to be an `if`/`else` clause instead of unless in this `if`.